src/components/MemberArea/EditProfile.js

```diff
@@ -57,7 +59,40 @@ export default class EditProfile extends Component {
       return;
     }
     this.setState({ disabled: true });
-    const updateMentorResult = await updateMentor(fromVMtoM(user));
+    const removeUserAvatar = (({
```

Comment:
This is a temporary fix to be able to submit the mentorship application pending when the api validation for avatar is updated or removed. This will be removed in the future

Reply:
I have removed that object as it was just a temporary fix for the backend avatar validation that wasn't removed